db/seeds/initial_seed.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+cohort2021 = Cohort.find_or_create_by!(start_year: 2021)
```

Comment:
nitpick, but can we have a `_` between words and numbers? `cohort_2021`?

Reply:
Rubocop told me off for that https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/VariableNumber

Reply:
Happy to disable the cop if we have consensus

Reply:
I remember not so long ago the cop was against having no underscores. I feel like they just dream up some of these cops

Reply:
I would try to configure it to `snake_case`. Maybe not disabling it all-together. 

And seeing if that breaks things too much :P